xml/installation-installation-ceph-alternative_supported_choice.xml

```diff
@@ -104,10 +104,12 @@
    <step>
     <para>
      Copy the <literal>entry-scale-kvm-ceph</literal> input model to the
-     <filename>~/helion/my_cloud/definition</filename> directory before you begin
+     <filename>~/openstack/my_cloud/definition</filename> directory before you 
+begin
      the editing process:
     </para>
-<screen>cp -r ~/helion/examples/entry-scale-kvm-ceph/* ~/helion/my_cloud/definition/</screen>
+<screen>cp -r ~/openstack/examples/entry-scale-kvm-ceph/* 
+~/openstack/my_cloud/definition/</screen>
```

Comment:
This breaks the command. If the plain-text version of this line is longer than 80 chars, just add a ` \`, otherwise remove the newline.